apps/web-embed/pages/PageWebEmbedPrimePurchase.tsx

```
@@ -0,0 +1,355 @@
+}
+
+function Spinner() {
+  return (
+    <svg
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      style={{
+        animation: 'spin 1s linear infinite',
+      }}
+    >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)